thoth/user_api/api_v1.py

```diff
@@ -323,7 +323,6 @@ def post_advise_python(
         except CacheMiss:
             pass
 
-    parameters["source_type"] = getattr(ThothAdviserIntegrationEnum, source_type.upper()) if source_type else None
     response, status = _do_schedule(parameters, _OPENSHIFT.schedule_adviser, output=Configuration.THOTH_ADVISER_OUTPUT)
```

Comment:
`schedule_adviser` accepts enum for `source_type`:

https://github.com/thoth-station/common/blob/088058ebd798e3909e4fd4c93f8b627545637823/thoth/common/openshift.py#L1632